src/experiments/distill/experiment_aug_mmlu_v0.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
```

Comment:
Could we have a dedicated file for each one of the models we want to experiement with?  Like `experiment_aug_mmlu_qwen3_235B.py`
Could you also compile a list of SOTA models we want to run this with?